Resources/config/fieldtypes.yml

```diff
@@ -1,7 +1,7 @@
 parameters:
     ezpublish.fieldType.eztags.class: Netgen\TagsBundle\Core\FieldType\Tags\Type
     ezpublish.fieldType.eztags.externalStorage.class: Netgen\TagsBundle\Core\FieldType\Tags\TagsStorage
-    ezpublish.fieldType.indexable.eztags.class: eZ\Publish\Core\FieldType\Unindexed
+    ezpublish.fieldType.indexable.eztags.class: Netgen\TagsBundle\Core\FieldType\Tags\SearchField
```

Comment:
This will also fail on 2014.11 because `eZ\Publish\SPI\FieldType\Indexable` interface does not exist.

Compiler pass that checks if interface exists and then sets this parameter to another one containing this class could solve this.

Compiler pass should probably only be enabled if the original value of the parameter is `eZ\Publish\Core\FieldType\Unindexed`, to allow for external overrides of the param.